src/services/user.service.js

```diff
@@ -0,0 +1,49 @@
+let users = [];
+
+function getRandomNumber() {
+  const min = 0;
+  const max = 100;
+
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
```

Comment:
you are reusing this function, don't you think there's a better way to optimize this?